hibernate-core/src/main/java/org/hibernate/metamodel/spi/RuntimeModelCreationContext.java

```diff
@@ -64,4 +67,11 @@ default MetadataImplementor getMetadata() {
 	Map<String, Generator> getGenerators();
 
 	GeneratorSettings getGeneratorSettings();
+
+	/*
+	 * Used by Hibernate Reactive
+	 */
+	default EntityMetamodel createEntityMetamodel(PersistentClass persistentClass, EntityPersister persister, RuntimeModelCreationContext creationContext) {
+		return new EntityMetamodel( persistentClass, persister, creationContext );
+	}
```

Comment:
```suggestion
	default EntityMetamodel createEntityMetamodel(PersistentClass persistentClass, EntityPersister persister) {
		return new EntityMetamodel( persistentClass, persister, this );
	}
```

Can just use `this` here.

Reply:
Good catch, solved